make/autoconf/lib-x11.m4

```diff
@@ -35,6 +35,7 @@ AC_DEFUN_ONCE([LIB_SETUP_X11],
     X_CFLAGS=
     X_LIBS=
   else
+    x_libraries_orig="$x_libraries"
```

Comment:
This needs to capture the value from `with_x` as well. Both are variants of user input.

Reply:
Hi Erik, I added a check in the with_x case.